zeek/assets/logs/zeek.yaml

```diff
@@ -1092,25 +1033,167 @@ pipeline:
       name: Remap `proto` attribute to `zeek.proto`
       enabled: true
       filter:
-        query: "@_path:(conn* OR notice OR traceroute OR dns OR dpd OR syslog) "
+        query: "@_path:(conn* OR notice OR traceroute OR dns OR dns_red OR dpd OR syslog) "
       processors:
         - type: attribute-remapper
           name: Map `proto` to `zeek.proto`
           enabled: true
-          meta:
-            tags: []
           sources:
             - proto
           sourceType: attribute
           target: zeek.proto
           targetType: attribute
           preserveSource: false
           overrideOnConflict: false
+    - type: pipeline
```

Comment:
It would be great to add a sample when you modify pipeline to cover this section. It will also help us understand what these properties mean and wether the facets you defined are really useful or not

Reply:
Added